src/meltano/core/plugin_install_service.py

```diff
@@ -36,7 +36,14 @@ class PluginInstallReason(str, Enum):
     """Plugin install reason enum."""
 
     ADD = "add"
+    CONFIG_TEST = "config test"
+    EL = "el"
+    ELT = "elt"
     INSTALL = "install"
+    INVOKE = "invoke"
+    RUN = "run"
+    SELECT = "select"
+    TEST = "test"
     UPGRADE = "upgrade"
```

Comment:
Should we collect all these newly-added install reasons under an aggregate `PREREQUISITE` reason, or something like that?

Reply:
Yeah, `PREREQUISITE` or something like `JIT` or similar seems like a good option

Reply:
[7f5c488](https://github.com/meltano/meltano/pull/8482/commits/7f5c48871975b4dfbdbb39526b28fa4a9a5bd292)